### PR TITLE
some minor fixes and renames.

### PR DESCRIPTION
**Pull request by lunfardo314**

### CODE REVIEW

File: tools/cluster/tests/advanced_inccounter_test.go [hunk 1]
```diff
@@ -20,74 +20,181 @@ import (
...
+		_, err = myClient.PostRequest(inccounter.FuncIncCounter)
+		require.NoError(t, err)
+	}
+
+	waitUntil(t, counterEquals(chain1, int64(numRequests)), sliceN(clusterSize), 15*time.Second)
+
+	printBlocks(t, chain1, numRequests+3)
+}
+
+func testAccessNodes4(t *testing.T) {
```
Comment (dessaya) [inline tools/cluster/tests/advanced_inccounter_test.go]: `testAccessNodes1` through `4` look almost identical, can't they be refactored into a single function?

**Review by dessaya — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -3,17 +3,20 @@ module github.com/iotaledger/wasp
...
 	github.com/ethereum/go-ethereum v1.10.3
 	github.com/iotaledger/goshimmer v0.7.2-0.20210628074845-4f90850164d1
-	github.com/iotaledger/hive.go v0.0.0-20210623095912-c1c6f098a6db
+	github.com/iotaledger/hive.go v0.0.0-20210625103722-68b2cf52ef4e
 	github.com/knadh/koanf v0.15.0
 	github.com/labstack/echo/v4 v4.1.13
 	github.com/mr-tron/base58 v1.2.0
 	github.com/pangpanglabs/echoswagger/v2 v2.1.0
 	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.7.0
```
Comment (dessaya) [inline go.mod]: What is this new dependency for?

File: tools/cluster/chain.go [hunk 3]
```diff
@@ -218,6 +218,29 @@ func (ch *Chain) BlockIndex(nodeIndex ...int) (uint32, error) {
...
+		})
+		if err != nil {
+			return nil, err
+		}
+		bi, err := blocklog.BlockInfoFromBytes(uint32(idx), res.MustGet(blocklog.ParamBlockInfo))
+		if err != nil {
+			return nil, err
+		}
+		ret = append(ret, bi)
+	}
```
Comment (jorgemmsilva) [inline tools/cluster/chain.go:240]: nitpick, start idx as int32:
```suggestion
	//nolint:unconvert
	for idx := uint32(blockIndex); idx >= 0; idx-- {
		res, err := cl.CallView(blocklog.FuncGetBlockInfo, dict.Dict{
			blocklog.ParamBlockIndex: codec.EncodeUint32(idx),
		})
		if err != nil {
			return nil, err
		}
		bi, err := blocklog.BlockInfoFromBytes(idx, res.MustGet(blocklog.ParamBlockInfo))
		if err != nil {
			return nil, err
		}
		ret = append(ret, bi)
	}
```

File: tools/cluster/tests/advanced_inccounter_test.go [hunk 4]
```diff
@@ -20,74 +22,176 @@ import (
...
+	return clu1, chain1
+}
+
+func sliceN(n int) []int {
+	ret := make([]int, n)
+	for i := range ret {
+		ret[i] = i
+	}
+	return ret
+}
```
Comment (jorgemmsilva) [inline tools/cluster/tests/advanced_inccounter_test.go]: I've added a function very similar to this recently:
https://github.com/iotaledger/wasp/blob/develop/tools/cluster/tests/util.go#L277-L283
should we create some "general utility" reusable functions package?

**Review by jorgemmsilva — APPROVED** [review]

File: tools/cluster/tests/advanced_inccounter_test.go (hunk 4 above)
Comment (lunfardo314) [reply tools/cluster/tests/advanced_inccounter_test.go]: good idea. I will put into general utils

**Pull request merged**